test/untitledSqlDocumentService.test.ts

```diff
@@ -0,0 +1,128 @@
+import * as TypeMoq from 'typemoq';
```

Comment:
yes not sure why. I see a permission error in travis job, might be related 

---
In reply to: [94176306](https://github.com/Microsoft/vscode-mssql/pull/569#discussion_r94176306) [](ancestors = 94176306)